content/components/recipes/form/arguments/PreferenceRollback.tsx

```diff
@@ -37,12 +38,15 @@ export default function PreferenceRollback(): ReactElement {
   };
 
   return (
-    <SelectField
-      searchable
-      data={rolloutData}
-      label="Rollout"
-      name="rolloutSlug"
-      renderMenuItem={rolloutMenu}
-    />
+    <>
+      <SelectField
+        searchable
+        data={rolloutData}
+        label="Rollout"
+        name="rolloutSlug"
+        renderMenuItem={rolloutMenu}
+      />
+      <ServerErrors field="arguments.slug" />
```

Comment:
This is redundant with what is done in `SelectField` but the field name is `slug` and not `rolloutSlug`

Reply:
Hm. Why is it `slug` here? The schema for rollback arguments [doesn't have a `slug` field](https://searchfox.org/mozilla-central/rev/ffdb6a4d934b3f5294f18cf0e1df618109ccb36b/toolkit/components/normandy/actions/schemas/index.js#112-124), it has `rolloutSlug`. Is the server returning errors on both?